osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -49,7 +49,7 @@ private void load(OsuConfigManager config, IDialogOverlay? dialogOverlay)
         {
             config.BindWith(OsuSetting.ReleaseStream, configReleaseStream);
 
-            if (updateManager?.CanCheckForUpdate == true)
+            if (updateManager?.CanCheckForUpdate == true && !RuntimeInfo.IsMobile)
```

Comment:
I guess we'll see if any android users immediately complain about this, can add back the button based on feedback I guess 🤷

Reply:
Actually this feels weird:

![osu! 2025-06-25 at 08 30 08](https://github.com/user-attachments/assets/0b175557-7124-4711-9149-39867f851107)

Will just add the button back.

Reply:
Please check e05c716d70af2cd72e8c60e7f8703bf4cd2d9a7e, did some minor refactoring for readability.